src/extensions/RhinoAssessmentHierarchyExtension.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace DNADesign\Rhino\Extensions;
+
+use DNADesign\Rhino\Pagetypes\RhinoAccount;
+use DNADesign\Rhino\Pagetypes\RhinoCapability;
+use DNADesign\Rhino\Pagetypes\RhinoCategory;
+use DNADesign\Rhino\Pagetypes\RhinoModule;
+use SilverStripe\ORM\DataExtension;
+
+class RhinoAssessmentHierarchyExtension extends DataExtension
+{
+
+	/**
+	 * Helper methods
+	 */
+	public function getModule()
+	{
+		return $this->owner->getParentOfType(RhinoModule::class);
```

Comment:
@alex-dna I don't know enough about this module - but if we are saying that they can use any nesting structure in the readme - I'm assuming that these could return null potentially? Is that handled?

Reply:
This methods are use on the RhinoAssessment, and the lack of results should be handled there.